Rubberduck.Refactorings/EncapsulateField/EncapsulateFieldModel.cs

```diff
@@ -11,20 +14,82 @@ public class EncapsulateFieldModel
     {
         public RubberduckParserState State { get; }
 
-        public Declaration TargetDeclaration { get; }
+        public EncapsulateFieldModel(RubberduckParserState state, QualifiedSelection selection)
+        {
+            State = state;
+            IList<Declaration> declarations = state.DeclarationFinder.UserDeclarations(DeclarationType.Variable).ToList();
```

Comment:
This is superfluous .  The first line of `FindVariable` is...
```
var items = declarations.Where(d => d.IsUserDefined && d.DeclarationType == DeclarationType.Variable).ToList();
```
...so it's already going to be filtered by type. This should be simply:

```
TargetDeclaration = state.DeclarationFinder.AllUserDeclarations.FindVariable(selection);
```